docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -129,13 +139,25 @@ For an end user, the UX to update the kubernetes version of the control plane an
 As an end user, I want to be able to easily scale up/down the number of replicas for each set of worker nodes in the cluster.
 - Currently, (for a cluster with 3 machine deployments) this is possible by updating these three different objects representing the sets of worker nodes in the pool. An easier user experience would be to update a single object to enable the scaling of multiple sets of worker nodes.
 
+#### Story 4 - Ability to customize individual clusters
+As a Cluster operator, I want to customize individual clusters simply by providing variables in the Cluster resource.
```

Comment:
can we be more specific here about what is "small deviations" may be going through a real world example that illustrate how the different personas benefit from this level of customization?

Reply:
Added an example, please let me know if that clarifies it.

tl;dr is basically if we don't have variables you have to create new ClusterClasses whenever you want to set a single field in any of the referenced templates differently. Looking at the cluster-templates we have in our providers (e.g. CAPA/CAPV), there are a lot of those.